docs/getting_started/login.md

```diff
@@ -16,17 +16,19 @@ If you need access to an existing project then ask the projects CloudFoundry mai
 ## Login to your project
 
 ```
-cf login -a https://api.system.staging.digital.gov.au
```

Comment:
You need to change digital.gov.au to cld.gov.au as well...

Reply:
Done!